framework/web/XmlResponseFormatter.php

```diff
@@ -50,7 +50,10 @@ class XmlResponseFormatter extends Component implements ResponseFormatterInterfa
      * @since 2.0.7
      */
     public $useTraversableAsArray = true;
-
+    /**
+     *  @var boolean mandatory addition of the tag of model name
+     */
+    public $modelTag = true;
```

Comment:
Should be renamed to `objectTags` since objects aren't necessary models in this case.